.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 💡General Feature Request
+about: Have a new idea/feature for Image-Searcher? Please suggest!
+title: '[FEATURE] <description>'
+labels: []
+assignees: []
+---
+
+**Description**
+
+Please describe the Feature in detail.
+
+**Expected Behavior**
+
+What happens instead?
```

Comment:
Here should be "What should happen?", following the pattern of the other configs